release.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euxo pipefail
+
+rm -rf release || exit 0;
```

Comment:
This is needed to remove the release dir, because I am using it later to clone the fresh to-be-released code.